etc/scripts/openhub_scraper.py

```diff
@@ -0,0 +1,28 @@
+# A script to extract openhub license names, their urls, and save it in a json file
+import json
+import urllib2
+import requests
+from bs4 import BeautifulSoup
+
+total_pages = 17
+license_dict = {}
+license_list = []
+
+try:
+	for i in range(1, total_pages + 1):
+		url = 'https://www.openhub.net/licenses?page=' + str(i)
+		page = urllib2.urlopen(url)
+		parsed_page = BeautifulSoup(page, 'html.parser')
+		all_licenses = parsed_page.find(id = 'license').select("table.table-condensed.table-striped.table")
```

Comment:
Please use only single quotes unless this is a docstring

Reply:
Oh damn, how did I miss that ! I'll correct that asap :)